package.json

```diff
@@ -20,10 +20,9 @@
   },
   "homepage": "https://github.com/demisto/content#readme",
   "dependencies": {
-    "@mdx-js/mdx": "^1.6.22",
```

Comment:
I think this was changed by accident. If not, sync your branch with your main branch *and* with our master branch, to get this synced between versions.

Reply:
Fixed.